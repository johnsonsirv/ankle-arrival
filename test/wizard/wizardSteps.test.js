/* eslint-disable react/jsx-props-no-spreading */
/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import WizardSteps from '../../src/wizard/wizardSteps';

function setup() {
  const enzymeWrapper = shallow(<WizardSteps />);

  return { props, enzymeWrapper };
}

describe('<WizardSteps /> rendering', () => {
  const { enzymeWrapper } = setup();

  it('should render correctly', () => {
    expect(enzymeWrapper).toMatchSnapshot();
  });
});
