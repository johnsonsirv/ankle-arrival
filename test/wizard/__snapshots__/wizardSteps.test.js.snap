// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WizardSteps /> rendering should render correctly 1`] = `ShallowWrapper {}`;
